dashboard/app/helpers/project_version_helper.rb

```diff
@@ -0,0 +1,7 @@
+module ProjectVersionHelper
+  PROJECT_TTL = 1.year
```

Comment:
If you're up for another commit, an explanation here of where this number comes from could be helpful! https://s3.console.aws.amazon.com/s3/management/cdo-v3-sources/lifecycle/view?region=us-east-1&id=365-day+expiration+of+old+versions

Totally optional, we could add this later.